libs/MailDB/include/MailDB/IMailDB.h

```diff
@@ -59,30 +59,19 @@ class IMailDB
     IMailDB(IMailDB&&) = delete;
     IMailDB& operator=(IMailDB&&) = delete;
 
-    // TODO: Viacheslav
     virtual void Connect(const std::string &connection_string) = 0;
     virtual void Disconnect() = 0;
     virtual bool IsConnected() const = 0;
 
-    // TODO: Denys
-    virtual void SignUp(const std::string_view user_name, const std::string_view hash_password) = 0;
```

Comment:
Parameter 'user_name' is const-qualified in the function declaration; const-qualification of parameters only has an effect in function definitions

Reply:
This has been implemented by @zikkend, please leave feedback in his pull-request about related issue

Reply:
I am not sure, but for me it seems more convinient to leave const in function declaration even though in this case 'const' is ignored by compiler